### PR TITLE
Implemented random BBO quote

### PR DESCRIPTION
**Pull request by spireken**
The build is in the book_view_window Dropbox folder.

### CODE REVIEW

File: Applications/spire/source/book_view_ui_tester/random_book_view_model.cpp [hunk 1]
```diff
@@ -132,7 +132,25 @@ connection RandomBookViewModel::connect_volume_slot(
...
-void RandomBookViewModel::update() {}
+void RandomBookViewModel::update() {
+  auto random_num = m_random_engine() % 3;
+  if(random_num == 0) {
+    return;
+  }
+  auto& bid_price = m_bbo.m_bid.m_price;
+  auto& ask_price = m_bbo.m_ask.m_price;
+  if(random_num == 1) {
+    if(boost::float64_t(bid_price) > 0.01) {
```
Comment (kranar) [inline Applications/spire/source/book_view_ui_tester/random_book_view_model.cpp]: Do not use floating points. Use Money::CENT to represent 1/100th of a dollar. If you need 10 cents you do 10 * Money::CENT. If you need units, then use Money::ONE. You can look up all the predefined units in the Money.hpp header file.

Comment (kranar) [inline Applications/spire/source/book_view_ui_tester/random_book_view_model.cpp]: Remove all instances of floats including boost::float64_t.

File: Applications/spire/source/book_view_ui_tester/random_book_view_model.cpp [hunk 2]
```diff
@@ -132,7 +132,25 @@ connection RandomBookViewModel::connect_volume_slot(
...
+      bid_price = bid_price - Money::CENT;
```
Comment (kranar) [inline Applications/spire/source/book_view_ui_tester/random_book_view_model.cpp]: Use the -= operator (x -= y);

**Comment by spireken** [comment]
I initially put the wrong build up, I ran it in Debug mode but put the RelWithDebugInfo build on Dropbox. That's fixed now. Timestamp is 3:47.

**Pull request merged**